src/rqt_py_console/py_console_widget.py

```diff
@@ -31,7 +31,7 @@
 # POSSIBILITY OF SUCH DAMAGE.
 
 import os
-import rospkg
+from ament_index_python.resources import get_resource
```

Comment:
The manifest should include an `exec_depend` on `ament_index_python`.

Reply:
done